users/stanrc85/config.h

```diff
@@ -7,3 +7,9 @@
   #define RGBLIGHT_LAYERS
   #define RGBLIGHT_SLEEP
 #endif
+
+#ifdef SNEAKBOX
+  #define INDICATOR_PIN_0 D7
+  #define INDICATOR_PIN_1 D6
+  #define INDICATOR_PIN_2 D4 
+#endif
```

Comment:
You could just do this: https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options

And you wouldn't need any extra code in your userspace, either.

Reply:
I'm not using the indicators for num lock or scroll lock though so I wasn't sure that helped me, unless I'm misunderstanding.